components/FilterBar/index.js

```diff
@@ -10,7 +10,7 @@ var _renderedFilterBar = null;
 class FilterBar extends Component {
   componentDidMount(): void {
     var scroller = this.refs.scroller.getDOMNode();
-    var width = [...scroller.children].reduce((num, c) => num + c.clientWidth, 0);
+    var width = [...scroller.children].reduce((num, c) => num + c.clientWidth, 0) + 50;
```

Comment:
looks like it was added to make sure the width was  enough, but just changing the starting value from 0 to 1 ensures the width is enough pixels for the filter